examples/profiling/README.md

```diff
@@ -64,21 +64,20 @@ This can be helpful in understanding the efficiency of various algorithms and th
 Use the pprof web interface to compare the performance of the 3 Fibonacci implementations.
 Refer to the [pprof docs] for more information about the web interface.
 
-[^1]:
-    Here "sampling" is in quotes because the profiler actually captures the call
+[^1]: Here "sampling" is in quotes because the profiler actually captures the call
```

Comment:
All other pages work fine with remark formatting... except this one. no mater what way I try to format it, it seems to decide footnotes are "not a thing" and formats to this (without the > quote prefix):

```markdown

> \[^1]: Here "sampling" is in quotes because the profiler actually captures the call
> stack at every cycle of program execution. Capturing a call stack on every
> cycle of execution is not done in most programs on physical CPUs for a few
> reasons:
> 
> ```
> - It would be cost prohibitive to do so for all but quite short program
>   executions.
> - Introducing such heavy profiling would actually alter the performance
>   characteristics in significant ways.
> 
> In zkVM execution, executions are generally short and all execution is
> synchronous and is not subject to any deviations in behavior due to
> measurement overhead.
> ```

```

unclear how to resolve that all... IMHO all MD files in the repo should use the same formatting & linting if possible

Reply:
yes, I have faced the same issue, I haven't quite found a way to fix this.

The only thing I was able to do is manually disable auto-save, revert he change and save again. Yes I agree it sucks.

Might revisit this today, maybe I'll have to write a custom remark plugin for this.

Reply:
Yeah, bummer on that behavior... have we filed a bug report yet?
I would advocate to get a md formatter for the  whole repo, and it is implicitly with the `.vscode` extension settings (if you use that IDE) 

But if that is much work to get configured to be sure it will not get in the way of writing docs (IMHO friction to get docs done is a _no no_ as it's usually a chore eng. don't want to do anyway... best devex for docs possible is a goal worth pursuing IMHO, even if it means a bit less polish)